Pod/Classes/URBNAlertStyle.m

```diff
@@ -153,17 +153,21 @@ - (id)copyWithZone:(NSZone *)zone {
     // UIColors
     styler.buttonBackgroundColor = [[UIColor alloc] initWithCGColor:self.buttonBackgroundColor.CGColor];
     styler.destructionButtonBackgroundColor = [[UIColor alloc] initWithCGColor:self.destructionButtonBackgroundColor.CGColor];
+    styler.destructiveButtonTitleColor = [[UIColor alloc] initWithCGColor:self.destructiveButtonTitleColor.CGColor];
     styler.buttonTitleColor = [[UIColor alloc] initWithCGColor:self.buttonTitleColor.CGColor];
     styler.backgroundColor = [[UIColor alloc] initWithCGColor:self.backgroundColor.CGColor];
     styler.titleColor = [[UIColor alloc] initWithCGColor:self.titleColor.CGColor];
     styler.messageColor = [[UIColor alloc] initWithCGColor:self.messageColor.CGColor];
     styler.alertViewShadowColor = [[UIColor alloc] initWithCGColor:self.alertViewShadowColor.CGColor];
     styler.blurTintColor = [[UIColor alloc] initWithCGColor:self.blurTintColor.CGColor];
+    styler.backgroundViewTintColor = [[UIColor alloc] initWithCGColor:self.backgroundViewTintColor.CGColor];
+    styler.errorTextColor = [[UIColor alloc] initWithCGColor:self.errorTextColor.CGColor];
 
     // UIFonts
     styler.titleFont = self.titleFont.copy;
     styler.messageFont = self.messageFont.copy;
     styler.buttonFont = self.buttonFont.copy;
+    styler.errorTextFont = self.errorTextFont.copy;
```

Comment:
Did you try removing `copy` on these? If you do stick with copy, you should use the method syntax rather than the property syntax.